internal/productcore/api_delete.go

```diff
@@ -1,11 +1,17 @@
 package productcore
 
-import "github.com/fastly/go-fastly/v10/fastly"
+import (
+	"context"
+
+	"github.com/fastly/go-fastly/v10/fastly"
+)
 
 // DeleteInput specifies the information needed for the Delete
 // function to perform the operation.
 type DeleteInput struct {
-	Client        *fastly.Client
+	Client *fastly.Client
```

Comment:
This looks like a formatting error... please be sure `go fmt` is checking these files too.

Reply:
Ahh, this is happening because of the comment, which is actually a bug in `go fmt`.